scripts/build/Dockerfile.openj9-ubuntu

```diff
@@ -22,12 +23,13 @@ RUN apt-install protobuf-c-compiler \
 	pkg-config \
 	iptables \
 	gcc \
-	maven
+	maven \
+	libdrm-dev
 
 COPY . /criu
 WORKDIR /criu
 
-RUN make
+RUN make mrproper && make -j $(nproc) CC="$CC"
```

Comment:
This commit has already been merged in the `criu-dev` branch;
`git rebase criu-dev` forllowed by `git push --force` would update your git branch and PR.

Reply:
OK